website/docs/r/monitor_action_group.html.markdown

```diff
@@ -166,6 +167,17 @@ The following arguments are supported:
 
 ---
 
+`event_hub_receiver` supports the following:
+
+* `name` - The name of the EventHub Receiver, must be unique within action group.
+* `event_hub_name` - (Required) The name of the specific Event Hub queue.
+* `event_hub_namespace` - (Required) The Event Hub namespace.
+* `subscription_id` - (Optional) The ID for the subscription containing this Event Hub.
```

Comment:
instead of 3 separate properties could we use the `event_hub_namespace.id` and parse these out?

Reply:
Hi @katbyte, I like the idea. Just to get a better understanding, you suggest that we just take the ID of the EventHub Namespace and then use the EventHub client to retrieve the rest of the required data within `expandMonitorActionGroupEventHubReceiver` ?

Reply:
I think I misunderstood you. You mean taking the resource id directly and parse subscription id, event hub name and namespace from the id, right? 

The issue is that the resource id of the namespace does not contain the name of the event hub. 

This is an example resource id:
`/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/group1/providers/Microsoft.EventHub/namespaces/namespace1`

Hence, we would still need the event hub name, and the subscription id is optional anyways.

Reply:
The ID of the actual eventhub within the namespace would work however:

`/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/group1/providers/Microsoft.EventHub/namespaces/namespace1/eventhubs/hub1`

Reply:
Hey @katbyte I added your suggestions, happy for any feedback

Reply:
sorry my mistake i swapped hub and namespace in my mind 😅  perfect thanks